library/src/main/java/com/github/javiersantos/piracychecker/LibraryUtils.java

```diff
@@ -144,4 +146,11 @@ static boolean isDebug(Context context) {
         return ((context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0);
     }
 
+    static boolean isConnectedToInternet(Context context) {
```

Comment:
`ConnectivityManager` will check if the user has WiFi or data enabled. If LuckyPatcher or any other patcher use an offline mode this method will return `true` because the device is connected to the Internet.

This method doesn't fix the issue reported by @franciscofranco , since the license checker will return "Allow" when the user is connected to the Internet and using a simple bypass by LuckyPatcher (offline mode).